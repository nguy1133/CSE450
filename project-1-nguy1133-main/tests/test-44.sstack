// ERROR: PUSH does not have a default argument.
PUSH
