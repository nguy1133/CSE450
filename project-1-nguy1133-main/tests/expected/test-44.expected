ERROR (line 2): Missing argument
